client/src/components/Sidebar/Feature/Feature.jsx

```diff
@@ -1,20 +1,78 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import styled from 'styled-components';
+
+const Descriptor = styled.div`
+font-size: 14px;
+display: flex;
+justify-content: space-between;
+align-items: center;
+`;
+
+const Border = styled.div`
+border-top: 1px solid #e7e7e7;
+font-size 14px;
+padding: 15px 15px 15px 0px
+`;
+
+const Title = styled.button`
+font-weight: bolder;
+font-size: 17px;
+line-height: 1.255;
+border: none; 
+padding: 0;
+background-color: transparent;
+outline: none;
+text-align: left;
+`;
+
+const Rating = styled.div`
+  color: #797979;
+  display: inline-block;
+`;
+
```

Comment:
I'd suggest new title for keep, can't really tell what it means on the page.

Reply:
Gotcha. Just modified this.

Reply:
I'll modify the other ones as well.